### PR TITLE
Mauve-2.3.1.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/e8c8c865a3c4ef19afd2 for a full test report.

File: easybuild/easyconfigs/m/Mauve/Mauve-2.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ alignments in the presence of large-scale evolutionary events such as rearrangement 
+ and inversion. Multiple genome alignment provides a basis for research into comparative 
+ genomics and the study of evolutionary dynamics. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://gel.ahabs.wisc.edu/mauve/downloads/']
+sources = ['%(namelower)s_linux_%(version)s.tar.gz']
+
+dependencies = [ ('Java', '1.7.0_21') ]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/Mauve/Mauve-2.3.1.eb]: possible to bump minor release tag? 

remove extra line at the end, overall ok here

Comment (fgeorgatos) [inline easybuild/easyconfigs/m/Mauve/Mauve-2.3.1.eb]: possibly to _80

File: easybuild/easyconfigs/m/Mauve/Mauve-2.3.1.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    'dirs': []
+}
+
+# add the installation dir to PATH
+modextrapaths = { 
+    'PATH': "",
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/Mauve/Mauve-2.3.1.eb]: remove extra line..

lgtm , put this in REVIEW

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**